src/main/java/uk/ac/ebi/embl/api/validation/submission/SubmissionValidator.java

```diff
@@ -110,14 +109,21 @@ SubmissionOptions mapManifestToSubmissionOptions(Manifest manifest) throws Valid
         }
         if (manifest.getSample() != null) {
             assemblyInfo.setBiosampleId(manifest.getSample().getBioSampleId());
```

Comment:
MasterSourceFeatureUtils sourceUtils = new MasterSourceFeatureUtils(); please delete  this if unused.